cypress/e2e/token-details.test.ts

```diff
@@ -55,14 +55,6 @@ describe('Token details', () => {
     if (cy.get('[data-cy="chart-header"]').contains('Price Unavailable')) {
       cy.get('[data-cy="missing-chart"]').should('exist')
     }
```

Comment:
If this is expected we could add a test to confirm
```suggestion
    }
    // Stats should not exist
    cy.get(getTestSelector('token-details-stats')).should('not.exist')
```